Trimble.Modus.Components/Hosting/AppConfig.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Trimble.Modus.Components.Hosting;
+
+public class AppConfig
```

Comment:
Can we have this named `ModusConfig`? Or something like that?

Reply:
Modified to ModusConfig